channels/in.m3u

```diff
@@ -364,17 +338,17 @@ https://news18india-lh.akamaihd.net/i/news18india_1@174951/index_5_av-p.m3u8
 #EXTINF:-1 tvg-id="OdishaTV.in" tvg-name="Odisha TV" tvg-country="IN" tvg-language="" tvg-logo="" group-title="",Odisha TV
 https://cdn.smartstream.video/smartstream-us/otvnewmbr/otvnewmbr.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="PopularScience.in" tvg-name="Popular Science" tvg-country="IN" tvg-language="English" tvg-logo="" group-title="Science",Popular Science
-https://a.jsrdn.com/broadcast/447912f76b/+0000/high/c.m3u8
+https://a.jsrdn.com/broadcast/447912f76b/+0000/c.m3u8
 #EXTINF:-1 tvg-id="PramayaNews7.in" tvg-name="Pramaya News7" tvg-country="IN" tvg-language="" tvg-logo="" group-title="News",Pramaya News7 (576p)
 https://live.mycast.in/encode/ee0c5a36ff5a7083ee044991974ad3ba.sdp/playlist.m3u8
 #EXTINF:-1 tvg-id="PratidinTime.in" tvg-name="Pratidin Time" tvg-country="IN" tvg-language="Hindi" tvg-logo="" group-title="News",Pratidin Time (360p)
 https://rtmp.smartstream.video/pratidintime/pratidintime/playlist.m3u8
 #EXTINF:-1 tvg-id="PrimeCanadaTV.in" tvg-name="Prime Canada TV" tvg-country="IN" tvg-language="Hindi" tvg-logo="http://mhdtvworld.com/wp-content/uploads/2019/07/PRIMECANADA.png" group-title="Entertainment",Prime Canada TV (720p)
 http://cdn27.live247stream.com/primecanada/247/primecanada/stream1/playlist.m3u8
 #EXTINF:-1 tvg-id="PUNJABTV.in" tvg-name="PUNJAB TV" tvg-country="IN" tvg-language="" tvg-logo="" group-title="",PUNJAB TV
-http://158.69.124.9:1935/5aabtv/5aabtv/chunklist_w642807911.m3u8
+http://158.69.124.9:1935/5aabtv/5aabtv/playlist.m3u8
 #EXTINF:-1 tvg-id="PunjabiZindabad.in" tvg-name="Punjabi Zindabad" tvg-country="IN" tvg-language="Hindi" tvg-logo="" group-title="",Punjabi Zindabad
-http://stream.pztv.online/pztv/tracks-v1a1/mono.m3u8
+http://stream.pztv.online/pztv/playlist.m3u8
```

Comment:
Infinite connection ecen from Indian proxy

Reply:
Ah... i'm gonna need to take a look at it.

Reply:
Found https://usb.bozztv.com/ssh101/PZLIVE/PZLIVE/playlist.m3u8

Thing is i'm blocked on a gateway timeout.

Reply:
Same here

Reply:
Approving as it is. If it's permanently down it will be removed by bot